x-pack/test/functional/apps/ml/anomaly_detection_jobs/single_metric_job.ts

```diff
@@ -70,16 +73,20 @@ export default function ({ getService }: FtrProviderContext) {
   }
 
   const calendarId = `wizard-test-calendar_${Date.now()}`;
+  const remoteName = 'ftr-remote:';
+  const indexPatternName = 'ft_farequote';
+  const indexPatternString = config.get('esTestCluster.ccs')
+    ? remoteName + indexPatternName
+    : indexPatternName;
 
   describe('single metric', function () {
     this.tags(['ml']);
     before(async () => {
-      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
-      await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
+      await esNode.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.createIndexPatternIfNeeded(indexPatternString, '@timestamp');
```

Comment:
In the `after` method we clean up this created index pattern / data view with `await ml.testResources.deleteIndexPatternByTitle('ft_farequote');`, so that one also needs to be adjusted to use `indexPatternString`.

Reply:
Fixed.